src/utils/ethereumUtils.ts

```diff
@@ -25,17 +28,29 @@ import {
   Linking,
   NativeModules,
 } from 'react-native';
+// @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
 import { ETHERSCAN_API_KEY } from 'react-native-dotenv';
 import { useSelector } from 'react-redux';
 import URL from 'url-parse';
+import {
+  DEFAULT_HD_PATH,
+  EthereumPrivateKey,
+  identifyWalletType,
+  WalletLibraryType,
+} from '../model/wallet';
```

Comment:
any reason to remove `} from '@rainbow-me/model/wallet';` ?

Reply:
the route was missing on the tsconfig.json file. just added it